backend/tests/services/knowledge/group_kb/conftest.py

```
@@ -0,0 +1,251 @@
+    )
+    test_db.add(member)
+    test_db.commit()
+    return member
+
+
+@pytest.fixture(scope="function")
+def group_kb(
+    test_db: Session,
+    test_group: Namespace,
+    test_group_owner: User,
+    group_owner_member: NamespaceMember,
```

Ruff (0.15.2)

[warning] Unused function argument: `group_owner_member`

(ARG001)